runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingTreesOverlay.java

```diff
@@ -87,6 +88,12 @@ private void renderAxes(Graphics2D graphics)
 		{
 			if (treeObject.getWorldLocation().distanceTo(client.getLocalPlayer().getWorldLocation()) <= 12)
 			{
+				// Check if this is a redwood tree at the farming patch
```

Comment:
This would be better written as a brief outlining that redwood trees at the farming guild work differently than any other tree, because they are multilocs (in order to be instanced for individual players) rather than a global object synced for all players.

Reply:
Updated, let me know if it should be expanded upon further